### PR TITLE
Godot: new recipe

### PR DESCRIPTION
**Pull request by miqlas**
Builds fine, but freaks out at linking stage. Help required.

### CODE REVIEW

**Comment by waddlesplash** [comment]
Paste of link errors?

**Comment by miqlas** [comment]
@waddlesplash : Does not compute.

**Comment by waddlesplash** [comment]
I mean, can you make a Pastebin or Gist or something of all the error messages you get while linking? You said it "freaks out" at the linking stage.

**Comment by miqlas** [comment]
@waddlesplash : here you go: https://hastebin.com/puzanifozo

**Comment by miqlas** [comment]
It seems hovewer it let to define not to use the shipped in-tree stuffs, it doesn't add the required LDFLAGS for the system libs.
But maybe i'm just wrong.

**Comment by waddlesplash** [comment]
Yep, that looks like what's going on. Seems to need at least `-lpng -lglew -lssl -lvorbis`.

**Comment by waddlesplash** [comment]
Works now?

**Comment by miqlas** [comment]
Yeah :)
xsi

**Comment by waddlesplash** [comment]
Oh, very nice.

**Comment by miqlas** [comment]
Btw, i got a demo working: 
xs-

File: dev-games/godot/godot-2.1.3.recipe [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+		#builtin_speex=no
+		#builtin_opus=no
+		#builtin_freetype=no
+		#builtin_squish=no
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir/Godot
+	mv bin/godot.haiku.tools* $appsDir/Godot/Godot
```
Comment (diversys) [inline dev-games/godot/godot-2.1.3.recipe]: If it's the only binary there is no need to put inside $appsDir/Godot/Godot

**Pull request merged**